src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+/**
+ * This class will represent an adding or deleting of a tag for a specific person that happened during that session.
+ */
+
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	
+	private String taggingLabel;
+	
+	public Tagging(String symbol,Person person, Tag tag){	
```

Comment:
Type String is not a good choice to maintain the action add/remove. Consider  a boolean or enum type.